v3/examples/websocket_echo/main.go

```diff
@@ -0,0 +1,181 @@
+/*
+In this example, the device sends an "echo" event whenever it receives a "doEcho"
```

Comment:
this is almost a copy of `examples/echo`.

Reply:
... which was basically a copy of the v2 echo... :P